src/utils/selectors/index.ts

```diff
@@ -1,3 +1,5 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils'
+
 export const effectCreator = `ClassProperty[value.callee.name='createEffect']`
```

Comment:
it looks like the `import` isn't used anymore.

```suggestion
export const effectCreator = `ClassProperty[value.callee.name='createEffect']`
```